clang/test/CodeGenSYCL/integration_header.cpp

```diff
@@ -27,36 +27,36 @@
 // CHECK: static constexpr
 // CHECK-NEXT: const kernel_param_desc_t kernel_signatures[] = {
 // CHECK-NEXT:   //--- _ZTSZ4mainE12first_kernel
-// CHECK-NEXT:   { kernel_param_kind_t::kind_std_layout, 4, 0 },
-// CHECK-NEXT:   { kernel_param_kind_t::kind_std_layout, 1, 4 },
-// CHECK-NEXT:   { kernel_param_kind_t::kind_std_layout, 4, 8 },
-// CHECK-NEXT:   { kernel_param_kind_t::kind_accessor, 4062, 12 },
-// CHECK-NEXT:   { kernel_param_kind_t::kind_accessor, 6112, 24 },
-// CHECK-NEXT:   { kernel_param_kind_t::kind_sampler, 8, 40 },
+// CHECK-NEXT:   { kernel_param_kind_t::kind_std_layout, 4, 0, (kernel_param_used[0] << 0)},
```

Comment:
These look right, just make sure you add the param_omit_table to the check-next lines.